### PR TITLE
🎨 improving category spending side-nav style

### PR DESCRIPTION
**Pull request by MatissJanis**
Related to: https://github.com/actualbudget/actual/issues/1497

Before:
Screenshot 2023-08-19 at 22 19 56

After:
Screenshot 2023-08-19 at 22 19 38

### CODE REVIEW

File: packages/desktop-client/src/components/reports/CategorySelector.tsx [hunk 1]
```diff
@@ -26,30 +26,29 @@ export default function CategorySelector({
...
-          type="bare"
-          style={{ padding: 4 }}
-          onClick={e => setUncheckedHidden(!uncheckedHidden)}
-        >
+        <Button onClick={() => setUncheckedHidden(state => !state)}>
           {uncheckedHidden ? (
             <>
               <Eye width={20} height={20} />
-              {'Checked'}
+              Show all categories
```
Comment (joel-jeremy) [inline packages/desktop-client/src/components/reports/CategorySelector.tsx:33]: Small nitpick: Maybe we should end both options with "categories" or both without?

Comment (MatissJanis) [reply packages/desktop-client/src/components/reports/CategorySelector.tsx:33]: Adding `categories` to the other label makes the button super stretched out. Which didn't look as good.

Got any other suggestions how we could call these labels?

Comment (joel-jeremy) [reply packages/desktop-client/src/components/reports/CategorySelector.tsx:33]: How about `Show unchecked` and `Hide unchecked`?

Comment (MatissJanis) [reply packages/desktop-client/src/components/reports/CategorySelector.tsx]: We can do that: https://github.com/actualbudget/actual/pull/1548/commits/fa7b25780e609ee79c67b603daf0a78d7a0cbaff

**Review by joel-jeremy — APPROVED** [review]
Looks good!

**Pull request merged**